src/sidebar/components/test/annotation-omega-test.js

```diff
@@ -33,17 +43,29 @@ describe('AnnotationOmega', () => {
   beforeEach(() => {
     fakeOnReplyCountClick = sinon.stub();
 
-    fakeQuote = sinon.stub();
+    fakeMetadata = {
+      isNew: sinon.stub(),
+      isReply: sinon.stub().returns(false),
+      quote: sinon.stub(),
+    };
+
+    fakePermissions = {
+      isShared: sinon.stub().returns(true),
+    };
```

Comment:
I'm a little wary of mocking very simple utility functions like these. Although it does make this test more of a pure unit test, it also increases the risk of a regression that goes undetected due to an integration issue, especially since we don't have type checking in place currently.

One way to guard against such issues is to make the return value conditional on the arguments (`stub().withArgs(...).returns(...)`).